src/traits.rs

```diff
@@ -36,19 +36,21 @@ pub trait Callback {
     fn call<T: ParserTrait>(cfg: Self::Cfg, parser: &T) -> Self::Res;
 }
 
-#[doc(hidden)]
```

Comment:
I'd say we should keep them hidden from the docs

Reply:
They do not appear in documentation the same, so inserting the attribute is useless. They only appear in the traits required by `Parser` struct, but we need to refactor the code more heavily to change that outocome